crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -197,6 +197,12 @@ impl AuthorityPerpetualTables {
             prev: None,
         }
     }
+
+    pub fn checkpoint_db(&self, path: &Path) -> SuiResult {
+        self.objects
```

Comment:
I think we need all tables?

Reply:
This is checkpointing all tables. We need to call checkpoint on any one column family to checkpoint the db. In the future, we can generate this function on the table struct through typed-store-derive macro (like some other functions we do today) which would let us invoke it like `self.checkpoint()` vs how we have to to do today `self.objects.checkpoint_db()`

Reply:
I see. Can you add comments to note this? It's not obvious reading it.